### PR TITLE
*: change golang.org/x/net/context to standard context

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What problem does this PR solve? 

Replace "golang.net/x/context" with standard "context"

It's a long time since https://github.com/pingcap/tidb/pull/2890, when many dependent libraries are still using "golang.net/x/context",
TiDB choose to use "golang.net/x/context"  for compatibility

I'm not sure the standard context works now, but it worth a try.

### What is changed and how it works?

run `GO111MODULE=on go fix ./...`

### Check List 

Tests 

 - No code

Side effects

 - Maybe compatibility issues

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/8579)

### CODE REVIEW

**Comment by ngaut** [comment]
Good job.

**Comment by ngaut** [comment]
LGTM

**Comment by lysu** [comment]
it seems this PR add some `cancel()` call addition to replace context, do we have test case cover new "cancel case" code path? :laughing:

**Comment by tiancaiamao** [comment]
@lysu  The error path, like:

```
ctx, cancel := context.WithCancel(...)
_, err := doSomething(ctx)
if err != nil {
    cancel() // The error path, if it's not add, golint will complaint 
}
```

File: cmd/benchdb/main.go [hunk 1]
```diff
@@ -21,12 +21,12 @@ import (
 	"strings"
 	"time"
 
+	"context"
```
Comment (zimulala) [inline cmd/benchdb/main.go]: Do we need to put this line to line 23?

Comment (tiancaiamao) [reply cmd/benchdb/main.go]: Done @zimulala

**Pull request merged**